<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exportDialog</class>
 <widget class="QDialog" name="exportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export to CSV</string>
  </property>
  <widget class="QLineEdit" name="exportPathInput">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>25</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="exportPathLabel">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>30</y>
     <width>74</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Path:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportPathBrowse">
   <property name="geometry">
    <rect>
     <x>447</x>
     <y>21</y>
     <width>87</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="exportPathButtonBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exportPathButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>exportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportPathButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>exportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
